app/controllers/api/v2/application_controller.rb

```diff
@@ -10,8 +10,14 @@ def authenticate
           email = email_from_token(token)
 
           @current_user = User.find_by("lower(email) = ?", email.downcase)
-
-          @current_user.present?
+          if @current_user.present?
+            Raven.user_context(id: @current_user.id)
+            true
+          else
+            # Once DFE Sign-In ID is passed in, add that to the Raven user
```

Comment:
current_user.sign_in_user_id 

thats the dfe sigh in id

Reply:
Not available in the backend, we're still waiting for @heidar to add it so that it's passed through in the payload.